src/interfaces/UserConfiguration.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * These are some of the common user configuration options used in Metals and needed for the editors.
+ *
+ *  - https://scalameta.org/metals/docs/integrations/new-editor#metals-user-configuration
+ */
+export const UserConfiguration = {
```

Comment:
I think you can use enum, with it exporting type shouldn't be necessary

Reply:
Changed!